ot/da.py

```diff
@@ -360,31 +367,35 @@ def sel(x):
     if log:
         log = {'err': []}
 
-    a, b = unif(ns), unif(nt)
+    a = nx.from_numpy(unif(ns), type_as=xs)
```

Comment:
use type_as of unif instead of converting it

Reply:
Added in [89addb6](https://github.com/PythonOT/POT/pull/343/commits/89addb6997f5c3df1e3f40625d893adb3ecca06d)